cli/tests/common/test_environment.rs

```diff
@@ -66,6 +66,8 @@ impl Default for TestEnvironment {
             r#"
 [template-aliases]
 'format_time_range(time_range)' = 'time_range.start() ++ " - " ++ time_range.end()'
+[git]
+write-change-id-header = false
```

Comment:
Is this tests stuff to avoid changing all the hashes? I think we should drop it and regenerate them all in another commit/PR, to avoid the test environment drifting from the actual defaults. (After all, maybe the change ID logic will cause bugs.)

Reply:
Yes, agreed. As I said in the commit message, I did it this way for now to avoid churn if we need to roll it back. Maybe some time after next release it's become clear enough that we're not going to roll it back, so then we can drop the config and update the tests. I don't feel strongly at all about exactly when we do it. I'm even fine with doing it in this PR if people don't mind the risk of updating it back and forth.

Reply:
Whoops, sorry for missing the commit message.

I think switching the tests over in a separate commit is certainly good, for reviewability. I would personally lean towards doing it in this PR and considering running the tests with the option on part of the testing we’d want to do over the next release period. Reverting should “just” be a matter of going over a tedious but automated snapshot diff, right? (IIRC the tests used to depend on exact commit hashes in places but I think that was reduced or removed?)

(My guess is that we won’t feel the need to revert this, since we’re already being quite conservative in a world where GitButler has been putting this stuff in headers for a while now.)

Reply:
I'm also pretty confident it will work. I went ahead and enabled the config in tests too.